generator/internal/retry_traits_generator.cc

```diff
@@ -53,7 +53,7 @@ Status RetryTraitsGenerator::GenerateHeader() {
     "\n"
     "/// Define the gRPC status code semantics for retrying requests.\n"
     "struct $retry_traits_name$ {\n"
-    "  static inline bool IsPermanentFailure(google::cloud::Status const& status) {\n"
+    "  static bool IsPermanentFailure(google::cloud::Status const& status) {\n"
```

Comment:
Consider sending this change in a separate PR as "preparing for clang-tidy 18" or something.